flatpak/build-flatpak.sh

```diff
@@ -1,5 +1,22 @@
 #!/usr/bin/env bash
 
+#   Copyright © 2019 by The qTox Project Contributors
```

Comment:
This file has an SPDX-License header and already a copyright notice, no need to add the full license too

Reply:
Changed to only update year range of existing copyright line, removed the full license text.